src/System.ValueTuple/tests/ValueTuple/UnitTests.cs

```diff
@@ -718,9 +718,12 @@ public static void ZeroTuples()
     {
         var a = ValueTuple.Create();
         Assert.True(a.Equals(new ValueTuple()));
+        Assert.Equal(0, a.CompareTo(new ValueTuple()));
 
         Assert.Equal(1, ((IStructuralComparable)a).CompareTo(null, DummyTestComparer.Instance));
         Assert.Throws<ArgumentException>(() => ((IStructuralComparable)a).CompareTo("string", DummyTestComparer.Instance));
+
+        Assert.Equal("(1, 2, 3, 4, 5, 6, 7, )", CreateLong(1, 2, 3, 4, 5, 6, 7, new ValueTuple()).ToString());
```

Comment:
The ", " at the end seems a bit odd, but I don't have a better suggestion (given it's a ```ValueTuple'8```, it'd be odd to omit the ValueTuple at the end as well).